### PR TITLE
[10.0] Set installable flag to True for l10n_ch_base_bank and l10n_ch_bank

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

**Comment by yvaucher** [comment]
Version in the manifest to change to 10.0.1.0.0

**Comment by yvaucher** [comment]
And please cherry-pick this change: https://github.com/OCA/l10n-switzerland/pull/281/commits/40784b8aff92c21489f95c27e959112ce06833be

File: l10n_ch_base_bank/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {'name': 'Switzerland - Bank type',
  'summary': 'Types and number validation for swiss electronic pmnt. DTA, ESR',
- 'version': '9.0.1.0.0',
+ 'version': '10.0.1.0.1',
```
Comment (leemannd) [inline l10n_ch_base_bank/__manifest__.py]: ? 10.0.1.0

**Review by leemannd — CHANGES_REQUESTED** [review]
Missing changes Version to V10 on "try me on runbot"

**Comment by grindtildeath** [comment]
@leemannd Thanks for the review

**Review by leemannd — APPROVED** [review]

**Comment by ghost** [comment]
Hi guys, is there any reason why this PR is not merged yet?
P.S. Don't forget to update the "Unported Addons" section in the base Readme of l10n-switzerland

**Comment by leemannd** [comment]
@BT-faebi I manly think that people that have the rights to merge it, also deserve some holidays ;) .

**Review by damdam-s — APPROVED** [review]
thanks

**Comment by yvaucher** [comment]
@BT-faebi No needs to change the README.md on root we have a bot for that.

Why not merged ? holidays ?

**Review by yvaucher — APPROVED** [review]

**Pull request merged**